extensions/blocks/social-previews/modal.scss

```diff
@@ -0,0 +1,80 @@
+/**
+ * Modal styles for Social Previews
+ */
+
+@import '~@automattic/color-studio/dist/color-variables';
```

Comment:
I think we may be able to get rid of this once we solve our issues with Color Studio and the recent update we made in #16312. 

See #16733 .

Reply:
Looking at the changes, it looks like we're importing this for a single variable: `$studio-gray-0` ([variable definition here](https://github.com/Automattic/color-studio/blob/master/dist/color-variables.scss#L6)). Given that this is a simple grey, we can probably use a Gutenberg grey instead for now. Thus swapping for `$light-gray-100` which is used as a background color in the donations block as well and is visually almost identical to `$studio-gray-0`.